mm/resources/ddl/category.sql

```diff
@@ -0,0 +1,42 @@
+-- PROCEDURE: get_sales_inventory_changes
+
+DROP PROCEDURE IF EXISTS `get_category_listing`;
+
+delimiter //
+
+CREATE PROCEDURE `get_category_listing`(
+    IN `store_id` INT
+
+)
+BEGIN
+
+SELECT
+   C.entity_id AS id,
+   C.entity_type_id,
+   C.attribute_set_id,
+   C.parent_id,
+   C.position,
+   C.level,
+   CCEI.value AS is_active,
+   CCEV.store_id,
+   CCEV.value AS name
+FROM catalog_category_entity C
+  INNER JOIN eav_entity_type EAVT ON C.entity_type_id = EAVT.entity_type_id
+  INNER JOIN (SELECT CEV.entity_id, CEV.value, CEV.store_id
```

Comment:
I don't understand. That would require temp tables to hold their value so I can join them on the main select statement. The subselect approach is more performant and cleaner. 
Unfortunately Magento's EAV design requires this kinds of things.